style.css

```diff
@@ -20,7 +25,7 @@ main{
   background-color: #f8f8f8;
   border-radius: 20px;
   width: 600px;
-  height: 360px;
+  height: fit-content;
```

Comment:
Is this fixing the issue of the part of data being displayed?

Reply:
yes